contrib/exporters/secadvisor/mod/transform_test.go

```diff
@@ -22,14 +22,70 @@ import (
 	"runtime"
 	"testing"
 	"time"
-
+	"io/ioutil"
 	cache "github.com/pmylund/go-cache"
-
+	"github.com/skydive-project/skydive/config"
 	"github.com/skydive-project/skydive/common"
-	"github.com/skydive-project/skydive/contrib/exporters/core"
+	//"github.com/skydive-project/skydive/contrib/exporters/core"
 	"github.com/skydive-project/skydive/flow"
 )
 
+const testConfig = `---
```

Comment:
you can remove all but the pipeline.transform section

Reply:
OK